plugin/src/main/java/net/mcjukebox/plugin/bukkit/managers/LangManager.java

```diff
@@ -62,6 +62,8 @@ private void addDefaults(){
 		addDefault("event.clientDisconnect", "&cYou disconnected from our audio server.");
 
 		addDefault("command.notOnline", "&c[user] is not currently online.");
+		addDefault("command.invalidUrl", "&cThe URL you entered is not valid. Please make sure that you entered the complete URL.");
+		addDefault("command.unexpectedUrl", "&6The URL you entered appears to be unsupported. Only direct file URLs are supported. Links to Spotify, YouTube, SoundCloud, etc. won't work.");
```

Comment:
Could we change this to: `&6Warning: The URL you provided does not have a recognised file extension and may fail to play. Links to files hosted on services like Spotify, YouTube and SoundCloud are unsupported.`

I think that reads a little nicer and the "warning" lets them know we'll still try it.